sys/dev/drm/bridges/tda19988/tda19988.c

```diff
@@ -261,6 +262,18 @@ __FBSDID("$FreeBSD$");
 #define	dprintf(fmt, ...)
 #endif
 
+/*
+ * The TDA19988 transmitter in Morello SoC r0p0 and r0p1 might not correctly
+ * detect hot-plugging due to incorrect voltage levels on the HPD pin.
+ *
+ * As a workaround for this issue, a user can set dev.drm.disable_hpd to enforce
+ * using a screen if the transmitter only receives an RxSense packet indicating
+ * a screen is attached, ignoring if it's actually able to receive HDMI input.
+ */
+static int tda19988_disable_hpd;
+module_param_named(disable_hpd, tda19988_disable_hpd, int, 0600);
```

Comment:
Ideally this would be namespaced to have tda(19988) in it

Reply:
I was hoping to place this tunable under `dev.tda.` or at leat `dev.drm.tda.` but drm uses the same sysctl parent [1] [2] (as we use `int` here) for all variables.

Would `dev.drm.tda_disable_hpd` be fine here? I'm thinking of `tda` rather than `tda19988` to match the name registered with `EARLY_DRIVER_MODULE()` - `tda`.

[1] https://github.com/CTSRD-CHERI/cheribsd/blob/dev/sys/dev/drm/drmkpi/include/linux/moduleparam.h#L41
[2] https://github.com/CTSRD-CHERI/cheribsd/blob/dev/sys/dev/drm/drmkpi/include/linux/moduleparam.h#L84